pyvenv.cfg

```diff
@@ -0,0 +1,3 @@
+home = /usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/bin
+include-system-site-packages = false
+version = 3.5.1
```

Comment:
You don't need this file in the repository.  You'll likely need to `git rm -f` it to make it go away.